src/kcc/api/viewsets.py

```diff
@@ -112,3 +121,72 @@ class ContactMomentViewSet(viewsets.ModelViewSet):
         "partial_update": SCOPE_KLANTEN_BIJWERKEN,
         "destroy": SCOPE_KLANTEN_ALLES_VERWIJDEREN,
     }
+
+
+class ObjectContactMomentViewSet(
+    CheckQueryParamsMixin,
+    mixins.CreateModelMixin,
+    mixins.DestroyModelMixin,
+    viewsets.ReadOnlyModelViewSet,
+):
+    """
+    Opvragen en verwijderen van OBJECT-CONTACTMOMENT relaties.
+
+    Het betreft een relatie tussen een willekeurig OBJECT, bijvoorbeeld een
```

Comment:
@sergei-maertens @joeribekker can it really be any type of object?

Reply:
Ja.